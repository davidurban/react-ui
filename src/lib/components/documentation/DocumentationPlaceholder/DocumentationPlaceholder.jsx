import PropTypes from 'prop-types';
import React from 'react';
import styles from './DocumentationPlaceholder.scss';

const DocumentationPlaceholder = props => (
  <div className={styles.root}>
    {props.text}
  </div>
);

DocumentationPlaceholder.propTypes = {
  text: PropTypes.string.isRequired,
};

export default DocumentationPlaceholder;
