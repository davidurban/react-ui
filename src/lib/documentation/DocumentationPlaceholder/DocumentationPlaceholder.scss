@import '../../shared/settings/colors';
@import '../../shared/tools/offset';

.root {
  padding: offset(3);
  border: 1px solid map-get($colors-grays, gray-200);
  background-color: map-get($colors-grays, gray-50);
}
