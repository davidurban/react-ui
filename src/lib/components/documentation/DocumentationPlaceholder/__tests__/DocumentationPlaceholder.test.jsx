import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import DocumentationPlaceholder from '..';

describe('rendering', () => {
  it('renders correctly mandatory props only', () => {
    const tree = shallow(<DocumentationPlaceholder text="text" />);

    expect(shallowToJson(tree)).toMatchSnapshot();
  });
});
