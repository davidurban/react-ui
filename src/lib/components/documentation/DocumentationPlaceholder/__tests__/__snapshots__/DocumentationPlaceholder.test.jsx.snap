// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders correctly mandatory props only 1`] = `
<div
  className="root"
>
  text
</div>
`;
